src/host/ft_host/API_MultipleInflightMessageTests.cpp

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "precomp.h"
+
+class MultipleInflightMessageTests
+{
+    BEGIN_TEST_CLASS(MultipleInflightMessageTests)
+    END_TEST_CLASS()
+
+    // This test is intended to make sure that we do not regress after the _handlePostCharInputLoop fix in OpenConsole:c0ab9cb5b
+    TEST_METHOD(WriteWhileReadingInputCrash)
+    {
+        BEGIN_TEST_METHOD_PROPERTIES()
+            TEST_METHOD_PROPERTY(L"IsolationLevel", L"Method") // Don't pollute other tests by isolating our potential crash and buffer resizing to this test.

```

Comment:
Huck a property in here that says this test needs appverif so we can detect those type by query when we figure out how to appverif in CI.